src/Umbraco.Web.UI.Client/src/less/components/editor/umb-editor.less

```diff
@@ -92,6 +93,11 @@
 
 .create-editor-sizes();
 
+.umb-editor--large {
+    max-width: 1600px;
+    width: calc(100% - 50px);
+}
+
```

Comment:
@abjerner does it need this?
Shouldn't it be handles as part of the `create-editor-sizes()` mixin?
https://github.com/umbraco/Umbraco-CMS/pull/11945/files#diff-30a9325d7dc3b4625468f745545e1fc0092ce04c1122c832b762a2ba8594f252R80

```
.umb-editor {
    &--@{size} {
        width: @value;
        will-change: transform;
        left: auto;
    }
}
```